features/dd-sdk-android-logs/api/dd-sdk-android-logs.api

```diff
@@ -1,3 +1,10 @@
+public final class com/datadog/android/log/BuildConfig {
```

Comment:
probably we don't need this to be generated for most of the library modules.

Reply:
True